### ISSUE TITLE
?

### ISSUE THREAD

**Issue by hasan1818666891** [shamhi/HamsterKombatBot#2437]
**I have one question!**

Is @shamhi got cheating card :grin:😁 ?

**Comment by ivan52605** [comment]
Мне кажется нет

**Comment by mirage2812** [comment]
я думаю да, у меня 99 акков из 100, работающих на запросах получили cheating card, один акк не авторизовал в сессию)) поэтому он остался

но код все равно крутой! многое взял как пример для дальнейшей работы

спасибо!

**Comment by Insomnia731** [comment]
> один акк не авторизовал в сессию)) поэтому он остался
Что это значит? На нем вообще не читерили или как?
Я например до этого кода просто выдернул bearer и накоалачивал через него, то есть сессии вообще не использовал, нужно было так и оставить, было бы с чем сравнить.
Вообще интересно, кто юзал скрипты и не отлетел в чём секрет.

**Comment by WeendVidente777** [comment]
> > один акк не авторизовал в сессию)) поэтому он остался
> 
> Что это значит? На нем вообще не читерили или как? Я например до этого кода просто выдернул bearer и накоалачивал через него, то есть сессии вообще не использовал, нужно было так и оставить, было бы с чем сравнить. Вообще интересно, кто юзал скрипты и не отлетел в чём секрет.

не жадничал. Ставил рандомные задержки между действиями, приблизительно похожие по таймингам на работу руками. Генерил ключи постепенно и рандомно. Не использовал больше шести аккаунтов с одного айпи. 
Хз. Больше не могу придумать в чем дело. 
Ну и игру, которая монетки давала вообще не делал. Слишком непонятно с ней было все, с фистингом и орал камшотом. Тогда же котключил первую минигру за ключи. 
БОльше идей нет почему мне повезло.

**Comment by Insomnia731** [comment]
> АПД. Там все равно дроп копеешный будет походу. Для меня пожалуй главная радость именно в том, что меня не забанили) Нормально заработает только тот, кто на стартовом пампе успеет все продать. Ну либо тот, кто ебался с фермой на 100+ аккаунтов.

То есть вообще ни одного акка не забанили и все они долбили запросами? Нормально, я грешил на фингерпринты и прокси, а что по проксям насколько левые?
Если так получается фармить нужно просто аккуратней, чтоб аномальных пиков по 100 ключей за день не было.

**Comment by WeendVidente777** [comment]
> > АПД. Там все равно дроп копеешный будет походу. Для меня пожалуй главная радость именно в том, что меня не забанили) Нормально заработает только тот, кто на стартовом пампе успеет все продать. Ну либо тот, кто ебался с фермой на 100+ аккаунтов.
> 
> То есть вообще ни одного акка не забанили и все они долбили запросами? Нормально, я грешил на фингерпринты и прокси, а что по проксям насколько левые? Если так получается фармить нужно просто аккуратней, чтоб аномальных пиков по 100 ключей за день не было.

Конкретно ачивки нет ни на одном. Но один вообще отлетел. Я в него залогиниться не могу через клиент, а через запросы там ответ приходит нестандартный. Но на этом аккаунте я все тестил, и слал кучу неправильных запросов. Думаю его фактически забанили, даже без ачивки. Все остальные норм, на который я уже протестеный и готовый скрипт запускал. 
Прокси не юзал, у меня только шесть аккаунтов было)

**Comment by Insomnia731** [comment]
> Конкретно ачивки нет ни на одном. Но один вообще отлетел. Я в него залогиниться не могу через клиент, а через запросы там ответ приходит нестандартный. Но на этом аккаунте я все тестил, и слал кучу неправильных запросов. Думаю его фактически забанили, даже без ачивки. Все остальные норм, на которых я уже протестеный и готовый скрипт запускал. Прокси не юзал, у меня только шесть аккаунтов было) В итоге по 4-5к монет на каждом посчитало.

Поздравляю, а у меня всего два акка было и оба отъехали, один долбил день и ночь тапалкой через термукс, а второй тестовый через какие только прокси не заходил от Ботсваны до Токио. Так и ачивки всё это время не было, повесили буквально часов 5 назад. Блин неприятно, типа как играл и проиграл.

А что за настройки в .env были, не выложишь кроме айди и хэша, интересно сравнить.

**Comment by molotov95** [comment]
8 Аккаунтов было на одном айпи, бота отключил за 10-13 дней, как карта появилась, и все получили дроп. В целом логично что перед дропом будут брить. Тут главное не жадничать)))

**Comment by Insomnia731** [comment]
> 8 Аккаунтов было на одном айпи, бота отключил за 10-13 дней, как карта появилась, и все получили дроп. В целом логично что перед дропом будут брить. Тут главное не жадничать)))

А в настройках этого скрипта всё по дефолту стояло как shamhi выставил?

**Comment by molotov95** [comment]
> > 8 Аккаунтов было на одном айпи, бота отключил за 10-13 дней, как карта появилась, и все получили дроп. В целом логично что перед дропом будут брить. Тут главное не жадничать)))
> 
> А в настройках этого скрипта всё по дефолту стояло как shamhi выставил?

Нет! Профиль настроек был такой:
MIN_AVAILABLE_ENERGY=100
SLEEP_BY_MIN_ENERGY=[1800, 3000]

AUTO_UPGRADE=True
MAX_LEVEL= 20
MAX_PRICE=50000000

BALANCE_TO_SAVE=50000000
UPGRADES_COUNT=10

MAX_COMBO_PRICE=20000000

APPLY_COMBO=True
APPLY_PROMO_CODES=True
APPLY_DAILY_CIPHER=True
APPLY_DAILY_REWARD=True
APPLY_DAILY_ENERGY=True
APPLY_DAILY_MINI_GAME=False

USE_RANDOM_MINI_GAME_KEY=True

USE_TAPS=True
RANDOM_TAPS_COUNT=[50, 200]
SLEEP_BETWEEN_TAP=[15, 40]

USE_RANDOM_DELAY_IN_RUN=True
RANDOM_DELAY_IN_RUN=[15, 40]

USE_RANDOM_USERAGENT=True

Еще изменял время задержки для получения ключей, в большую сторону

**Comment by Insomnia731** [comment]
> Еще изменял время задержки для получения ключей, в большую сторону

О спасб, то есть ключи всё таки получал, а все грешили на ключи.
Ну и кстати логично "бота отключил за 10-13 дней", я бы тоже так сделал, да общее время игры было всего неделя, недавно вкатился.
Моя теория на данный момент, что есть определенные пики и тот кто фармит как не в себя и палится на фоне других юзеров, вот тот и попадает под раздачу.

**Comment by Tsunami43** [comment]
0 проксей, вместо этого раскидывал по 20 акков по сервакам, принты разные и задержки везде где можно = 0 отлетевших аккаунтов, хз как так=) аптайм прерывался на пару секунд для пула обновления

**Comment by WeendVidente777** [comment]
> > Конкретно ачивки нет ни на одном. Но один вообще отлетел. Я в него залогиниться не могу через клиент, а через запросы там ответ приходит нестандартный. Но на этом аккаунте я все тестил, и слал кучу неправильных запросов. Думаю его фактически забанили, даже без ачивки. Все остальные норм, на которых я уже протестеный и готовый скрипт запускал. Прокси не юзал, у меня только шесть аккаунтов было) В итоге по 4-5к монет на каждом посчитало.
> 
> Поздравляю, а у меня всего два акка было и оба отъехали, один долбил день и ночь тапалкой через термукс, а второй тестовый через какие только прокси не заходил от Ботсваны до Токио. Так и ачивки всё это время не было, повесили буквально часов 5 назад. Блин неприятно, типа как играл и проиграл.
> 
> А что за настройки в .env были, не выложишь кроме айди и хэша, интересно сравнить.

у меня свой скрипт, на ноде.

**Comment by treyne** [comment]
> > один акк не авторизовал в сессию)) поэтому он остался
> 
> Что это значит? На нем вообще не читерили или как? Я например до этого кода просто выдернул bearer и накоалачивал через него, то есть сессии вообще не использовал, нужно было так и оставить, было бы с чем сравнить. Вообще интересно, кто юзал скрипты и не отлетел в чём секрет.

Да, есть, юзал свои скрипты, частично код брал из этого проекта, бан не дали.
